main.tf

```diff
@@ -78,27 +78,15 @@ resource "null_resource" "run_ansible_from_bootstrap_node_to_install_dcos" {
 
   # with a check for the case that we are on Azure checking the metadata url,
   # as long as cloud-init is not supported on the CentOS and RHEL based azure images
+  provisioner "file" {
+    destination = "/tmp/bootstrap-docker-install.sh"
+
```

Comment:
extra line?

Reply:
with only two options we could leave it out sure